charts/prometheus-adapter/values.yaml

```diff
@@ -217,6 +217,9 @@ podLabels: {}
 # Annotations added to the pod
 podAnnotations: {}
 
+# Annotations added to the deployment
+deploymentAnnotations: {}
```

Comment:
I've personally been naming this key as `workloadAnnotations` in my personal charts.